core/src/main/java/greencity/controller/HabitController.java

```diff
@@ -285,4 +277,14 @@ public ResponseEntity<CustomHabitDtoResponse> updateCustomHabit(@PathVariable Lo
         return ResponseEntity.status(HttpStatus.OK)
             .body(habitService.updateCustomHabit(request, habitId, principal.getName(), image));
     }
+
+    /**
+     * Handles IllegalArgumentException.
+     *
+     * @return Error message "Bad Request".
+     */
+    @ExceptionHandler({IllegalArgumentException.class})
```

Comment:
Can we add an exception handler for IllegalArgumentException here https://github.com/ita-social-projects/GreenCity/blob/dev/core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java ?

In this case, we will cover the same exception for other endpoints